### ISSUE TITLE
Win 2012 R2 server remote desktop reconnect needed after each login

### ISSUE THREAD

**Issue by jacotec** [Ylianst/MeshCentral#3610]
This is reproducible on every single W2012R2 server I manage:

When you connect to the server which displays the login screen, then send CTRL-ALT-DEL and enter the password, MeshCentral remote desktop freezes after login:

<img width="808" alt="2022-02-06_20-31-51" src="https://user-images.githubusercontent.com/3321655/152698138-e7aa1a2f-8379-4c07-86b4-131a994e0371.png">

It's needed to disconnect and reconnect the remote desktop to get to the desktop after login. The "update desktop" icon does not work.

**Comment by krayon007** [comment]
Yeah, I noticed this earlier with this version of windows. Normally the kvm process will exit when it gets a permission exception, and the parent process will respawn into the new session. But for some reason this version of windows does not return an exception in this case, the screen scrape succeeds but returns black image data. I meant to try to workaround this by hooking the user login/log off event to force respawn the process.

**Comment by si458** [comment]
this is a duplicate issue, will tag it later when i find it.
fix at moment is to simply, disconnect and reconnect and ur good to go!
